### ISSUE TITLE
Improve documentation about rectangular clustering - which rectangles get merged and which don't

### ISSUE THREAD

**Issue by alexbfree** [zooniverse/aggregation#143]
http://zooniverse-aggregation.readthedocs.org/en/latest/rectangle_aggregation.html is a good start, but for me it leaves a lot of questions:

_“referring to the same thing”_
It's not clear to me how the aggregation code can tell whether rectangles are referring to the same thing or not. I think it would be good to add an example where there are 3 distinct regions on the subject - two of which overlap.
this would let you talk about the easy case (non-overlapping entity where all rectangles refer to the same thing) and the harder case (two overlapping entities, each of which with their own collection of rectangles).

I think it's important in the writing to distinguish between the multiplicity of rectangles due to one person marking several areas, vs the multiplicity of rectangles for many people each drawing one rectangle pertaining to a particular region... It's not always clear which you are talking about.

I think a photographic example might make this clearer.
For example:
![asg0018jt5](https://cloud.githubusercontent.com/assets/1473244/14850117/cec737ca-0c6e-11e6-802a-952f5e00f164.jpg)
which aggregates to
![asg0018jt5](https://cloud.githubusercontent.com/assets/1473244/14850126/d94d4770-0c6e-11e6-96b3-41933ea4759d.jpg)
(this is subject 1925322 in Snapshot Serengeti: Computer Vision)
If you used an example like this, that could help make it clearer between overlapping _animals_ and overlapping _rectangles_.

_the triangle_
It's really hard to follow how this triangle relates to the rectangle diagram above. I think this could be spelled out a bit, to make the page flow better.

_general context of this page_
this page isn't very clear about what the general technique being described is. It uses the phrase "clustering rectangles" without really explaining what that means. For example, are we talking about identifying which rectangles should be merged for a single cluster? Or are we talking about identifying several clusters from many rectangles? Are we talking about overlapping _opinions_ or overlapping _entities_ (or both)? All of the above? Maybe break it down into the separate aims/challenges, and how each one is being tackled...

_ending_
The page ends rather abruptly. It would be good to have some sort of conclusion - the final bit about cliques feels a bit of non-sequitur without saying what that actually means or referring back to the context of the problem in hand.

**Comment by ggdhines-zz** [comment]
"I think it's important in the writing to distinguish between the multiplicity of rectangles due to one person marking several areas, vs the multiplicity of rectangles for many people each drawing one rectangle pertaining to a particular region... It's not always clear which you are talking about." right now there is no distinction. I've updated the rectangle documentation to help explain that

**Comment by ggdhines-zz** [comment]
I like the example images @alexbfree used above - I'll add them to the documentation page and update the graph example based on them.

**Comment by ggdhines-zz** [comment]
"clustering rectangles" - "are we talking about identifying which rectangles should be merged for a single cluster? Or are we talking about identifying several clusters from many rectangles?" Yes to both - they are both part of what clustering is. I'll add a link to the wikipedia article on clustering.

**Comment by ggdhines-zz** [comment]
ending - yeah definitely just stopped. Added a few last sentences

**Comment by ggdhines-zz** [comment]
@alexbfree "overlapping opinions or overlapping entities" - what is an opinion vs. what is an entity?

**Comment by alexbfree** [comment]
Well two rectangles can overlap because two different people marked the same "thing"(animal/entity) in different places.
Or they can overlap because the actual entities(animals in this case) overlap and even a single user would draw those rectangles as overlapping.
It would be really good if wherever we mention "overlap" we make it clear which we are talking about.
